### DISCUSSION TITLE
Question About Custom Domain

### DISCUSSION THREAD

**Discussion by strawbs1989** [community/community#119997] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

just wondering with the custom domain, can i add coolvibes-reloaded.com onto this 
domain https://strawbs1989.github.io/coolvibes-reloaded/  

if so what are you're nameservers and records

**Comment by CopperEagle** [comment]
Hey there @strawbs1989 :)

At first I thought this is not possible... now editing in that it actually is. [Here](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages) is the documentation for how to do it.

Before the edit, I pointed out there is another solution: There are sites that allow you to create a redirection. This means they redirect people visiting coolvibes-reloaded.com to your other domain. One such page is fly.io which should be free, but I have never tested it myself.

[Here](https://support.glitch.com/t/tutorial-get-your-glitch-project-on-a-domain/5579) is a tutorial how to use it. Follow it from **step 2**. The tutorial is actually for redirecting towards a glitch.me server rather than a github page. However, the procedure should be pretty much the same. As you can see in the tutorial, the "Where is your site hosted" window during that process features a Github pages option too.

In any case, please let us know about your experience with it!
